src/database/redis.config.js

```diff
@@ -0,0 +1,5 @@
+import redis from 'redis';
```

Comment:
Can you move this file in `src/config/` because it looks to me more like a configuration and it would be better for us to keep all configs in `src/config/` unless you are obliged to put it elsewhere.

Reply:
I put it in the database because it deals with redis database but also the configurations of sequelize are in that database directory so I figured everything database related should be in one directory. Plus the name of the file is descriptive as to not be confused for something else.